.github/workflows/label-messager.yml

```diff
@@ -0,0 +1,17 @@
+name: "Label Messager"
+on:
+  issues:
+    types: ["labeled"]
+  schedule:
+    # Run pull request triage every 5 minutes. Ideally, this would be on
+    # "labeled" types of pull request events, but that doesn't work if the pull
+    # request is from another fork. For example, see
+    # https://github.com/actions/labeler/issues/12
+    - cron: '*/5 * * * *'
```

Comment:
Could this be bundled into one of the preexisting cron flows to avoid spawning more actions? The current "actions" tab of the repo already has a fair bit of noise and I think it'd be best to move these to one place.

Reply:
Done